### ISSUE TITLE
Configtool: runtime errors/exceptions should create some visual

### ISSUE THREAD

**Issue by Traumflug** [Traumflug/Teacup_Firmware#243]
While trying to reproduce this bug: https://github.com/Traumflug/Teacup_Firmware/issues/242, it occurs to me that there is no visual feedback on runtime errors. In this case, the file fails to save, but the GUI gives zero notice of this. A user can see such errors only when running from the command line.

Ideally, Configtool would catch all exceptions and forward them to the user somehow.

**Comment by drf5n** [comment]
Yes.  It took me a while to even notice the file saving menu on my Mac, and then I thought I was doing something wrong since I couldn't seem to save/change the printer name.  Then I noticed the errors in the terminal.
